install/kubernetes/helm/subcharts/tracing/templates/deployment-zipkin.yaml

```diff
@@ -0,0 +1,57 @@
+{{ if eq .Values.provider "zipkin" }}
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-zipkin
+  labels:
+    app: zipkin
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: zipkin
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.zipkin.hub }}:{{ .Values.zipkin.tag }}"
```

Comment:
As mentioned elsewhere, might be more consistent to have hub represent the org, and add image `{{ .Values.zipkin.hub }}/zipkin:{{ .Values.zipkin.tag }}`

Reply:
done.